packages/expo-dev-client/e2e/ios/AppDelegate.m

```diff
@@ -0,0 +1,113 @@
+#import "AppDelegate.h"
+
+#import <React/RCTBridge.h>
+#import <React/RCTBundleURLProvider.h>
+#import <React/RCTRootView.h>
+#import <React/RCTLinkingManager.h>
+
+#import <UMCore/UMModuleRegistry.h>
+#import <UMReactNativeAdapter/UMNativeModulesProxy.h>
+#import <UMReactNativeAdapter/UMModuleRegistryAdapter.h>
+#import <UMCore/UMModuleRegistryProvider.h>
+
+@import EXDevMenu;
+#include <EXDevLauncher/EXDevLauncherController.h>
+
+
+@interface DevMenuDetoxTestInterceptor : NSObject<DevMenuTestInterceptor>
+
+@end
+
+@implementation DevMenuDetoxTestInterceptor
+
+- (BOOL)isOnboardingFinishedKey
+{
+  return YES;
+}
+
+- (BOOL)shouldShowAtLaunch
+{
+  return NO;
+}
+
+@end
+
+@interface AppDelegate () <RCTBridgeDelegate>
+
+@property (nonatomic, strong) EXModuleRegistryAdapter *moduleRegistryAdapter;
+@property (nonatomic, strong) NSDictionary *launchOptions;
+
+@end
+
+@implementation AppDelegate
+
+- (BOOL)application:(UIApplication *)application didFinishLaunchingWithOptions:(NSDictionary *)launchOptions
+{
+  [DevMenuTestInterceptorManager setTestInterceptor:[DevMenuDetoxTestInterceptor new]];
+
+  self.moduleRegistryAdapter = [[EXModuleRegistryAdapter alloc] initWithModuleRegistryProvider:[[EXModuleRegistryProvider alloc] init]];
+  self.launchOptions = launchOptions;
+  self.window = [[UIWindow alloc] initWithFrame:[UIScreen mainScreen].bounds];
+
+   EXDevLauncherController *controller = [EXDevLauncherController sharedInstance];
```

Comment:
I think it would also be worth having a test case with the expo-updates integration. Do you think that should be a separate app?

Reply:
I was thinking about it. I think it will be cleaner to have another app for updates integration. However, the CI job will take more time to finish then. So for now, we should add it to the existing one.